test/find.js

```diff
@@ -40,7 +40,7 @@ test('simple path', t => {
   t.is(result.code, 0);
   t.truthy(result.indexOf('test/resources/find/.hidden') > -1);
   t.truthy(result.indexOf('test/resources/find/dir1/dir11/a_dir11') > -1);
-  t.is(result.length, 11);
+  t.is(result.length, 12);
```

Comment:
Why is this increased?

Reply:
Because there is actually a new entry: the link to the dir2 folder.
Unfortunately git did not add it in the first commit, now I pushed a new commit that adds it.

Reply:
264470c91100948905108f19995bb0b193fee982

Reply:
Ah, I see it now. Looks good!